docs/deployment.md

```diff
@@ -0,0 +1,91 @@
+
+This document details the process we use to deploy Page Shot to our stage and production environments.
+
+## Overview of schedule ##
+
+(you can read more below)
+
+- Thursday at 12pm PST the train is cut, all code is in. Style tweaks can be made after this point.
```

Comment:
I think we could do it later in the day than this.  2pm Pacific?  Also assuming we'll respect daylight savings, PST isn't technically correct